src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -1671,8 +1671,9 @@ def _parse_aoi(data):
     serializer.is_valid(raise_exception=True)
     area_of_interest = serializer.validated_data.get('area_of_interest')
     wkaoi = serializer.validated_data.get('wkaoi')
+    layer_overrides = data.get('layer_overrides', {})
```

Comment:
It is a bit redundant since line 1662 has already got this object. I didn't pass it as a parameter to `_parse_aoi` since there are other places (e.g. analyze) that uses this function. Not very sure what the best practice is here